CODE_OF_CONDUCT.md

```diff
@@ -31,17 +33,16 @@ Examples of unacceptable behavior by participants include:
 * Other conduct which could reasonably be considered inappropriate in a
  professional setting
 
-## Our Responsibilities
+## Enforcement Responsibilities
 
 Project maintainers are responsible for clarifying the standards of acceptable
 behavior and are expected to take appropriate and fair corrective action in
-response to any instances of unacceptable behavior.
+response to any behavior that they deem inappropriate,
```

Comment:
Just be aware here and throughout, it looks like the original text was using 80char lines. Don't care either way, but want to be consistent in the one doc. I'd just follow the decision of the original markdown which appears to be 80 char lines (see https://www.contributor-covenant.org/version/2/0/code_of_conduct/code_of_conduct.md )

Reply:
Yeah, I don't care either way, which is why I'd defer to the decision upstream had made (not sure if their reasons, but I'd guess it wasn't related to punch-cards). That said, another way to look at consistency is that all *our* docs (regardless of where they came from) use the same. +1 for natural lines in that case.

Reply:
Agree. Keep with natural lines